spec-0004/index.md

```diff
@@ -91,26 +91,36 @@ Complete examples of how projects implement this in their CI setup are linked in
 #### Process for Adding New Projects
 
 After someone creates an issue on https://github.com/scientific-python/upload-nightly-action
-requesting access to upload wheels a human has to respond to that request.
+requesting access to upload wheels a human/admin has to respond to that request.
+
+Admins are people from the community who are ideally not part of the same organizations nor
+project. This is to prevent malicious activities from a given group of actors and ensure a
+diverse and healthy community. Adding new people to the list requires of admins requires at
+least an issue to be openned.
 
 We want to be open to projects uploading wheels but at the same time need to perform some
 amount of due dilligence before giving people access. This is because once a user is given
-access they could upload wheels for any project. We assume that people are not malicious
-and we can see from the logs who misbehaved after the fact.
+access they benefit the wide exposure of Scientific Python and could take this to their
+advantage to publish malicious packages.
 
 Once you have established who the person is and that they represent the project they want
 to upload wheels for ask the person to create an account on https://anaconda.org and tell
-you the username.
-
-You can then add them to the `scientific-python-nightly-wheels` organisation on anaconda.org.
-Let the user know that they have been added and that they can create a access token at
-https://anaconda.org/scientific-python-nightly-wheels/settings/access. The token should
-only have the "Allow uploads to Standard Python repositories" and
-"Allow write access to the API site" scope. It should use the project name as the token name.
-
-You can also add new people to https://anaconda.org/scientific-python-nightly-wheels when they
-contact you privately, but it would be good if an issue is created as part of that. This helps
-with keeping track of who did what when.
+you the username. We suggest projects to have a least 2 representatives and ideally they
+should not be part of the same organization to ensure a diverse community.
```

Comment:
what do we mean by organization here? I think this is worth clarifying, because a project could also be interpreted as an organization?

Reply:
I mean like you belong to the same project, company or OSS community.

Reply:
Ok, so the sentence before this says that projects have at least 2 representatives, but that they should not be part of the same project, company, or OSS community? Maybe we could say something like 

> We suggest that projects have at least 2 representatives. When considering representatives, we suggest that projects nominate individuals that do not have significant community, organizational, or employer overlap with existing representatives to ensure that we have a diverse community.

Reply:
I added your suggestion thanks!